src/drt/src/dr/FlexDR_init.cpp

```diff
@@ -2760,6 +2802,40 @@ void FlexDRWorker::route_queue_update_from_marker(
   }
 }
 
+void FlexDRWorker::getRipUpNetsFromMarker(frMarker* marker,
+                                          set<drNet*>& nets,
+                                          frCoord bloatDist)
+{
+  // if shapes don't overlap routeBox, ignore violation
+  if (!getRouteBox().intersects(marker->getBBox())) {
+    bool overlaps = false;
+    for (auto& s : marker->getAggressors()) {
+      if (std::get<1>(s.second).intersects(getRouteBox())) {
```

Comment:
future: we really need a struct with proper names from getAggressors as this is quite unreadable.